compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/node/Owner.kt

```diff
@@ -276,7 +276,6 @@ internal interface Owner : PositionCalculator {
      * Android, this schedules view tree layout observer callback to be invoked for the underlying
      * platform view hierarchy.
      */
-    // TODO: Move to InteropViewContainer
```

Comment:
https://youtrack.jetbrains.com/issue/CMP-7382/Refactor-InteropViewHolder-InteropViewContainer-in-commonMain